tests/integrations/huggingface/cassettes/test_huggingface_inference_client/test_huggingface_table_question_answering_async.yaml

```
@@ -1,79 +1,408 @@
 interactions:
-- request:
-    body: null
+    body: ''
     headers:
+      accept:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)